packages/nextjs/types/abitype/abi.d.ts

```diff
@@ -7,15 +7,3 @@ declare module "abitype" {
     AddressType: AddressType;
   }
 }
-
-declare module "viem/node_modules/abitype" {
-  export interface Register {
-    AddressType: AddressType;
-  }
-}
-
-declare module "wagmi/node_moudles/abitype" {
```

Comment:
looks like we don't need to declare module for wagmi at all. No abitype in `wagmi`'s deps, and typo here (`node_moudles`) and everything worked fine

Reply:
lol it was hiding all along until now!